app/code/Magento/InventoryApi/Test/_files/stock_rollback.php

```diff
@@ -7,7 +7,10 @@
 
 use Magento\InventoryApi\Api\StockRepositoryInterface;
 use Magento\TestFramework\Helper\Bootstrap;
+use Magento\InventorySales\Model\ResourceModel\DeleteSalesChannelToStockLink;
 
+$deleteLink = Bootstrap::getObjectManager()->get(DeleteSalesChannelToStockLink::class);
```

Comment:
We don't need to remove `non-existed-website` because according to our logic should be the validation error
Also, links will be removed automatically on website delete
So we can remove this code

Reply:
done. removed all changes to the fixture. was create coz when test was writen no validator existed